backend/LexBoxApi/GraphQL/ProjectMutations.cs

```diff
@@ -339,6 +339,26 @@ await dbContext.ProjectUsers.Where(pu => pu.ProjectId == input.ProjectId && pu.U
         return dbContext.Projects.Where(p => p.Id == input.ProjectId);
     }
 
+    [Error<NotFoundException>]
+    [Error<DbError>]
+    [UseMutationConvention]
+    [UseFirstOrDefault]
+    [UseProjection]
+    public async Task<IQueryable<DraftProject>> DeleteDraftProject(
+        Guid draftProjectId,
+        IPermissionService permissionService,
+        LexBoxDbContext dbContext)
+    {
+        permissionService.AssertCanManageProject(draftProjectId);
+
+        // Draft projects are deleted immediately, not soft-deleted
+        var deletedDraftCount = await dbContext.DraftProjects.Where(dp => dp.Id == draftProjectId).ExecuteDeleteAsync();
+        if (deletedDraftCount == 0)
+        {
+            throw NotFoundException.ForType<DraftProject>();
+        }
+        return dbContext.DraftProjects.Where(p => p.Id == draftProjectId);
```

Comment:
This will always return an empty list right?

Reply:
Yeah, I'd say we query the draft out so we can return it here, as an object.

Reply:
Done.